_pages/clients/r.md

```diff
@@ -160,6 +160,18 @@ Now you've seen how to upload a local data file, check if a file exists in a dat
 
 For more methods on how to get a file using the Data API from a data collection go to the [API Specification](http://docs.algorithmia.com/#getting-a-file).
 
+{% if site.enterprise %}
+## Publishing Insights
+
+Inference-related metrics (a feature of [Insights](../../algorithmia-enterprise/insights) can be reported via using the `report_insights` method of the Algorithmia client.
+
+{% highlight r %}
+# report Insights
+# TBD R sample code for this - https://algorithmia.atlassian.net/browse/INSIGHTS-31
+client$report_insights(...)
```

Comment:
Replace `$` with `.`.

Reply:
Other method calls documented in this file use `$` as the object / method separator. Should this deviate from those examples?

Reply:
Oh my bad, thought this was a typo. :joy: